pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -21,14 +21,16 @@ import (
 	"fmt"
 	"time"
 
+	"k8s.io/apimachinery/pkg/api/meta"
```

Comment:
unneeded change.

Reply:
Used at line 140 and 151 (meta.IsNoMatchError)

Reply:
It's a duplicate of line 22.